### PR TITLE
Strict version 20200902

### PR DESCRIPTION
**Pull request by jkeenan**
This p.r. does two things.  It updates all files such that they would be able to run in a "strict-by-default" environment.  It increments the VERSION number in pod inside `Install.pm`.

Thank you very much.
Jim Keenan

### CODE REVIEW

**Comment by karenetheridge** [comment]
If perl 5.6 (or greater) is a minimum requirement already, then we should add `use strict` to all files, now that this can be done safely.

**Comment by atoomic** [comment]
Note that `our` is already used in multiple locations view https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/blob/master/lib/ExtUtils/Install.pm#L12 for example
so enforcing `strict` seems fine

**Comment by jkeenan** [comment]
Pull request has been updated per suggestions in review.

No changes made to t/lib/Test/* files, in deference to issue https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/issues/24 and p.r. https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/pull/25.

Please re-review then merge as appropriate.

Thank you very much.
Jim Keenan

File: t/lib/Test/Builder/Module.pm [hunk 1]
```diff
@@ -3,9 +3,9 @@ package Test::Builder::Module;
 use Test::Builder;
 
 require Exporter;
-@ISA = qw(Exporter);
+our @ISA = qw(Exporter);
 
-$VERSION = '0.03';
+our $VERSION = '0.03';
 
 use strict;
```
Comment (toddr) [inline t/lib/Test/Builder/Module.pm:10]: Not the end of the world but this strict should probably move up.

Comment (jkeenan) [reply t/lib/Test/Builder/Module.pm:10]: Is this not one of the files that would be removed in your p.r. https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/pull/25?

**Comment by Leont** [comment]
> If perl 5.6 (or greater) is a minimum requirement already, then we should add use strict to all files, now that this can be done safely.

`use strict` doesn't have a 5.6 dependency, strict has been with us since 5.000. It's `use warnings` that requires 5.6.

File: t/lib/Test/Builder/Module.pm (hunk 1 above)
Comment (toddr) [reply t/lib/Test/Builder/Module.pm:10]: It is.

Comment (mohawk2) [reply t/lib/Test/Builder/Module.pm:10]: #25 now merged, this is sorted :-)

**Comment by mohawk2** [comment]
Could this be rebased now #25 is in? Then I'd assume it'll be ready for merge.

**Comment by jkeenan** [comment]
> Could this be rebased now #25 is in? Then I'd assume it'll be ready for merge.

Rebased on master.  For consistency with other files, I moved `use strict` up to top of file in two t/*.t files.

**Review by toddr — APPROVED** [review]

**Pull request merged**